packages/wrangler/src/config/environment.ts

```diff
@@ -743,6 +743,21 @@ export interface EnvironmentNonInheritable {
 		  }
 		| undefined;
 
+	/**
+	 * Binding to Cloudflare Images
+	 *
+	 * NOTE: This field is not automatically inherited from the top level environment,
+	 * and so must be specified in every named environment.
+	 *
+	 * @default `{}`
+	 * @nonInheritable
+	 */
+	images:
```

Comment:
I know we are not doing docs yet, but when we do we should not forget to update https://developers.cloudflare.com/workers/wrangler/configuration/#non-inheritable-keys as well. A lot of folks tend to forget